src/aktualizr_primary/primary_secondary_registration_test.cc

```diff
@@ -43,6 +43,7 @@ TEST(PrimarySecondaryReg, SecondariesMigration) {
     // prepare storage the "old" way:
     storage->storeDeviceId("device");
     storage->storeEcuSerials({{primary_serial, primary_hwid}, {secondary_serial, secondary_hwid}});
+    storage->saveSecondaryInfo(secondary_serial, "IP", PublicKey());
```

Comment:
You're cheating here, this part of the the database did not exist before this feature.
The code that's removed is supposed to handle this case.

Some explanations are in this commit message https://github.com/advancedtelematic/aktualizr/commit/72b47fb306b96571d67b2fc8af721d4d65279a1e but we should put them in a comment probably.

Reply:
Ah, alright, I see. Yes, I've missed this case.
Why can't we still make it from one place though? If we call it from Initializer as a step separate from `initEcuRegister`, it would cover this case as well, wouldn't it?

Reply:
Yes, that sounds like a better plan. I think it could work.